src/app-layout/visual-refresh-toolbar/drawer/global-drawers.tsx

```diff
@@ -29,12 +29,13 @@ export function AppLayoutGlobalDrawersImplementation({
             (drawer.preserveInactiveContent && openDrawersHistory.current.has(drawer.id))
         )
         .map(drawer => {
-          openDrawersHistory.current.add(drawer.id);
+          setTimeout(() => openDrawersHistory.current.add(drawer.id), 0);
```

Comment:
I'm not 100% sure if it's the best possible way to implement it. the point of this condition is to add drawerId to the open history only after first render. Technically it's possible to expose a callback like `onMount` from `AppLayoutGlobalDrawer` and set value inside of it